zarr/tests/test_meta.py

```diff
@@ -601,7 +601,7 @@ def test_metadata3_exceptions():
     required = ["zarr_format", "metadata_encoding", "metadata_key_suffix", "extensions"]
     for key in required:
         meta = copy.copy(_default_entry_point_metadata_v3)
-        meta.pop("zarr_format")
+        meta.pop(key)
```

Comment:
I hope I haven't missed something here. Otherwise, what would be the point of the `key` iterator?

Reply:
Wow 🤯